modules/javafx.graphics/src/main/java/javafx/css/Stylesheet.java

```diff
@@ -294,12 +315,14 @@ public static Stylesheet loadBinary(URL url) throws IOException {
     }
 
     /**
-     * Convert the .css file referenced by source to binary format and write to destination.
-     * @param source the JavaFX .css file to convert
-     * @param destination the file to which the binary version is written
-     * @throws IOException if any IO error occurs.
-     * @throws IllegalArgumentException if either parameter is null, if source and destination are the same,
-     * if source cannot be read, or if destination cannot be written.
+     * Convert the css file referenced by source to binary format and write to destination.
```

Comment:
"source" should be `{@code source}` since it's a method argument. Same for "destination".